#Type of message
string type

#Joint Command/Telemetry 
float32 joint_1
float32 joint_2
float32 z_axis
float32 joint_4
float32 gripper